collector/manager/metrics_collector_manager.go

```diff
@@ -211,9 +224,9 @@ func (mcm *MetricsCollectorManager) declareMetricDefinitions(metricDetails []col
 
 		// put all the tags together for the full list of tags to be applied to this metric definition
 		allMetricTags := tags.Tags{}
-		allMetricTags.AppendTags(globalTags)   // global tags are overridden by...
+		allMetricTags.AppendTags(metricTags)   // metric tags are overriden by
 		allMetricTags.AppendTags(endpointTags) // endpoint tags which are overridden by...
-		allMetricTags.AppendTags(metricTags)   // metric tags.
+		allMetricTags.AppendTags(globalTags)   // global tags
```

Comment:
You sure you want to reverse the order here? I can't think of why, but for some reason I am thinking I remember saying to myself, "Oh good, the metrics override everything - I need that because..." but I can't remember why :)